src/routers/city.js

```diff
@@ -0,0 +1,11 @@
+const {Router} = require('express');
+const router = Router();
+const {getCities, postCity} = require('../controllers/city-controller');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
const {getCities, postCity} = require("../controllers/city-controller");
```